app/Leaf/Site.php

```diff
@@ -82,7 +82,7 @@ private function checkPath(): array
     {
         $vars = array(':site_path' => $this->match);
         $sql = 'SELECT `site_path`, `site_uploads`, `portal_database`, `orgchart_path`,
-                    `orgchart_database`
+                    `orgchart_database`, `launchpadID`, `decommission`
                 FROM `sites`

```

Comment:
Couple things:
- This highlights another prerequisite that didn't come up before. The "sites" table is apparently missing from the [resources database boilerplate](https://github.com/department-of-veterans-affairs/LEAF/blob/master/docker/mysql/db/boilerplate/resource_database_boilerplate.sql), and should be added. All Request Portals should have identical **core** codebases and database schemas.
- LEAF-4204 outlines a specific naming convention for the new column: "decommissionTimestamp". This should not be changed unless there's a compelling reason.

Reply:
Ok, these have been addressed and will be pushed soon.